linux/keyman-config/build.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
+# shellcheck source=resources/build/build-utils.sh
+. "${THIS_SCRIPT%/*}/../../resources/build/build-utils.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+################################ Main script ################################
+
+builder_describe \
+  "Build keyman-config." \
+  "clean" \
+  "configure" \
+  "build" \
+  "test" \
+  "install                   install artifacts" \
+  "uninstall                 uninstall artifacts" \
+
+builder_parse "$@"
+
+cd "$THIS_SCRIPT_PATH"
+
+builder_describe_outputs \
+  build "keyman_config/standards/lang_tags_map.py"
```

Comment:
Either:
```suggestion
  build "./keyman_config/standards/lang_tags_map.py"
```

or

```suggestion
  build "/linux/keyman-config/keyman_config/standards/lang_tags_map.py"
```

Second is preferable because it is faster :grin: